scripts/globals/crafting.lua

```diff
@@ -115,21 +114,55 @@ function getTestItem(player, npc, craftID)
 
     switch (npc:getName()): caseof
     {
-        ["Abd-al-Raziq"] = function (x)     TItemList = TI_Alchemy end,
-        ["Peshi_Yohnts"] = function (x)     TItemList = TI_Bonecraft end,
-        ["Ponono"] = function (x)             TItemList = TI_Clothcraft end,
-        ["Piketo-Puketo"] = function (x)     TItemList = TI_Cooking end,
-        ["Thubu_Parohren"] = function (x)     TItemList = TI_Fishing end,
-        ["Reinberta"] = function (x)         TItemList = TI_Goldsmithing end,
-        ["Faulpie"] = function (x)             TItemList = TI_Leathercraft end,
-        ["Mevreauche"] = function (x)         TItemList = TI_Smithing end,
-        ["Ghemp"] = function (x)             TItemList = TI_Smithing end,
-        ["Cheupirudaux"] = function (x)     TItemList = TI_Woodworking end,
+        ["Abd-al-Raziq"] =      function (x)    TItemList = TI_Alchemy end,
+        ["Peshi_Yohnts"] =      function (x)    TItemList = TI_Bonecraft end,
+        ["Ponono"] =            function (x)    TItemList = TI_Clothcraft end,
+        ["Piketo-Puketo"] =     function (x)    TItemList = TI_Cooking end,
+        ["Thubu_Parohren"] =    function (x)    TItemList = TI_Fishing end,
+        ["Reinberta"] =         function (x)    TItemList = TI_Goldsmithing end,
+        ["Faulpie"] =           function (x)    TItemList = TI_Leathercraft end,
+        ["Mevreauche"] =        function (x)    TItemList = TI_Smithing end,
+        ["Ghemp"] =             function (x)    TItemList = TI_Smithing end,
+        ["Cheupirudaux"] =      function (x)    TItemList = TI_Woodworking end,
```

Comment:
Just look at that beautiful alignment. <3

Reply:
Indeed. Very nice.

I am curious however if this really needs to be a switch? There's no actual function code happening here, and although we C++ types like to think of switches as fast, Lua has no actual switch and there's a fair bit of boilerplate that has to happen to set this 'simulacrum of a switch' up for what could be an easy table index.

Reply:
:smiley: